yaml/builders/image-build-push.yaml

```diff
@@ -32,17 +32,23 @@
                 image_tags+=$(docker image inspect "$image_id" --format '{{{{range $i, $tag := .RepoTags}}}}{{{{$tag}}}} {{{{end}}}}')
             done
 
+            # All images are built in format quay.io/centos7/<image_name>
+            # image_tags looks like:
+            # quay.io/centos7/python-38-centos7:1 quay.io/centos7/python-38-centos7:20201125-f9402d2 quay.io/centos7/python-38-centos7:latest quay.io/centos7/python-38-centos7:raw
             for image in $image_tags; do
               case $image in
                 *:raw|*:squashed) continue ;;
               esac
 
               # By default images are now tagged into localhost/ registry
-              image_name=${{image/#localhost\//}}
-              image_name=quay.io/${{image_name}}
-              echo "Pushing ${{image_name}}"
+              # Image name for Docker Hub
+              # hub_namespace is docker.io/centos and we need to get from image only image name
+              image_name={hub_namespace}/\${{image##*/}}
+              echo "Pushing ${{image}} into Quay Hub"
```

Comment:
Isn't there supposed to be a `docker tag` call before `docker push` as well?

Reply:
Good point. By default, all our images are built like `quay.io/centos7/<image_name>`. Before this change, images were built with tag `centos/<image_name>`. Therefore docker tag was needed.

Reply:
Thanks, makes sense.